scripts/README.md

```diff
@@ -4,3 +4,26 @@
 
 To import all the application jsons from the application store for a specific `round_id` you run (from project root):
 `python -m scripts.import_from_application --roundid={your round id}`
+
+## Populate Location Data
+
+This script does the following:
+
+1. Extracts postcode from the 'address of asset' field in the application form
+1. Sends these postcodes to postcodes.io to retrieve further location information, eg. country
+1. Updates the database, populating `assessment_records.location_json_blob` with a json blob containing the location details
+1. Writes the retrieved location data out to a csv file at the specified location.
+
+To run locally:
+
+        ./scripts/populate_location_data.py --fund_id 47aef2f5-3fcb-4d45-acb5-f0152b5f03c4 --round_id c603d114-5364-4474-a0c4-c41cbf4d3bbd --update_db True --write_csv True --csv_location ./locations.csv
+
+To run on paas, execute the following (starts a task using the app context etc)
+
+        cf run-task funding-service-design-assessment-store-dev --command "./scripts/populate_location_data.py --fund_id 47aef2f5-3fcb-4d45-acb5-f0152b5f03c4 --round_id c603d114-5364-4474-a0c4-c41cbf4d3bbd --update_db True --write_csv True --csv_location /tmp/locations.csv && cat /tmp/locations.csv" --name populate_location
```

Comment:
Will these commands use `python -m` now? If so please can we update?

Reply:
Yes, good shout